01-introduction.Rmd

```diff
@@ -55,9 +54,6 @@ non-spatial attribute data are inevitably interwoven with geometry data, and Geo
 The latter communicates that this is a book about using R as a GIS, to perform spatial operations on *geographic data* [@bivand_applied_2013].
 However, the term GIS conveys some connotations (see Table \@ref(tab:gdsl)) which simply fail to communicate one of R's greatest strengths:
 its unparalleled console-based ability to seamlessly switch between geographic and non-geographic data processing, modeling and visualization tasks.
```

Comment:
well, unparalleled is maybe a bit too much. For instance, you can do stuff like this in Python as well